### ISSUE TITLE
wrong size of conv1d in CBHG, Post-processing net

### ISSUE THREAD

**Issue by msobhan69** [Kyubyong/tacotron#13]
See https://github.com/Kyubyong/tacotron/blob/master/networks.py#L94 , according to Table 1 of the paper, the size of 2'th layer of Conv1D projections should be 80.

**Comment by tozangezan** [comment]
Here. (networks.py; L112~)

     
 `dec = conv1d(dec, hp.embed_size//2, 3, scope="conv1d_1") # (N, T', E)`
      `  dec = normalize(dec, type="bn", is_training=is_training, 
                            activation_fn=tf.nn.relu)`
 `dec = conv1d(dec, hp.embed_size//2, 3, scope="conv1d_2") # (N, T', E)`
I think first method should be conv1d(dec, hp.embed_size, 3, scope="conv1d_1") and the second should be conv1d(dec, 80, 3, scope="conv1d_1")

**Comment by Kyubyong** [comment]
You guys are right. I've changed. Thanks.

**Comment by candlewill** [comment]
@Kyubyong The dimension is updated. However, this change leads to a new issue in [highway net](https://github.com/Kyubyong/tacotron/blob/master/modules.py#L278):

```
outputs = H * T + inputs * C
```

The dimension of `inputs` and `C` does not match. The `inputs` tensor has shape _[N, T, W]_, _W=80_, while the `C` tensor has shape _[N, T, num_units]_, _num_units=128_.

My suggestion is to add a dense layer before highway layer. This dense layer transform the `inputs` into a tensor of shape _[N, T, num_units]_.

**Comment by Kyubyong** [comment]
You're right candlewill. But I don't see any particular reason why we should make things complicated, so I'll just change the output units of the second conv1d layer to 128 for simpliicity.
